src/njoy21/input/MODER.hpp

```diff
@@ -37,6 +37,13 @@ class MODER {
             card3List->emplace_back( is, this->card1.nout );
           }
           card3List->pop_back();
+          for( unsigned int i = 1; i < card3List->size(); i++ ){
```

Comment:
Look at [LEAPR::Card8::Alpha](https://github.com/njoy/NJOY21/blob/master/src/njoy21/input/LEAPR/Card8/Alpha.hpp#L26) for a way to do this using the standard library. That would be preferred over this `for` loop.